src/main/java/de/milchreis/uibooster/UiBooster.java

```diff
@@ -25,35 +24,42 @@
     private final UiBoosterOptions options;
 
     public UiBooster() {
-        this(new UiBoosterOptions(UiBoosterOptions.Theme.DARK_THEME));
+        this(new DefaultUiBoosterOptions());
     }
 
     public UiBooster(UiBoosterOptions options) {
-        this.options = options == null ? new UiBoosterOptions() : options;
-
-        if (options.getTheme() != null) {
-            try {
-                if (options.getTheme() == UiBoosterOptions.Theme.DARK_THEME) {
-                    // Little hack to start working on linux
-                    UIManager.getFont("Label.font");
-                    UIManager.setLookAndFeel(new DarculaLaf());
-
-                } else if (options.getTheme() == UiBoosterOptions.Theme.OS_NATIVE) {
-                    UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-
-                } else if (options.getTheme() == UiBoosterOptions.Theme.SWING) {
-                    UIManager.setLookAndFeel(new MetalLookAndFeel());
-
-                } else if (options.getTheme() == UiBoosterOptions.Theme.DEFAULT) {
-                    UIManager.setLookAndFeel(UIManager.getLookAndFeel());
-                }
+        this.options = options;
+        try {
+            UIManager.setLookAndFeel(options.getLookAndFeel());
+        }
+        catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Upon second look this looks suspect and requires understanding of how swing works in the background